docs/tutorials/managing_multiple_workspaces.rst

```diff
@@ -0,0 +1,61 @@
+=============================
+Managing multiple workspaces
+=============================
+.. _tutorial-managing-multiple-workspaces:
+
+Before learning how to manage multiple workspace with RosTeamWorkspace be sure that you have set everything up as described in this :ref:`tutorial <tutorial-setting-up-rtw>`.
+
+Also be sure that you opened a new terminal after you setup RosTeamWorkspace to be configured permanently.
+
+
+First setup a new workspace called ``ws_rolling_ros2c_demos`` inside a ``~/workspace`` folder.
+
+.. code-block:: bash
+
+   setup-ros-workspace ~/workspace/ws_rolling_ros2c_demos rolling
+
+When asked for confirmation just press <ENTER>.
+After a workspace is created open a new terminal and execute ``_ws_rolling_ros2c_demos`` alias for sourcing your new workspace. You can then switch to the new sourced workspace with ``rosd``.
+Now you can use :ref:`aliases <uc-aliases>` to interact with your workspace.
+Those can be used out of any folder you are in.
+
+Let's now add a test package into your workspace.
+
+1. Make sure your workspace is sourced by executing ``_ws_rolling_ros2c_demos``. Then enter the source folder using ``rosds`` alias.
+2. Clone ``ros2_control_demos`` repository for testing either:
+
+.. code-block:: bash
+
+   git clone git@github.com:ros-controls/ros2_control_demos.git # SSH
+   # or if no ssh key is setup:
+   git clone https://github.com/ros-controls/ros2_control_demos.git # HTTPS
+
+3. Go to the base of your workspace:
+
+.. code-block:: bash
+
+   rosd
+
+4. Install dependencies to be sure we got everything:
+
+.. code-block:: bash
+
+    rosdep install --ignore-src --from-paths src -y -r
```

Comment:
@mamueluth do you think that we should add also an alias for this? It is often used command in almost the same format and it is a bit longer. We know the `src` folder from the workspace definition. Or would this be an overkill?

Reply:
#60